### PR TITLE
Added short blog post for 2.3.0 release

### PR DESCRIPTION
**Pull request by merlimat**
### Motivation

Added blog post with brief explanation of release highlights

### CODE REVIEW

**Comment by merlimat** [comment]
> * python state 

That one is there ;)

> * debezium integration

This one I unintentionally missed

**Comment by merlimat** [comment]
(actually it was there only on my local)

**Comment by sijie** [comment]
Awesome! @merlimat

File: site2/website/blog/2019-02-20-Apache-Pulsar-2-3-0.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+The Apache Pulsar PMC is happy to announce the release of Pulsar 2.3.0. This
+is the result of huge effort from the community, with over 480 commits and
+a long list of new features, general improvements and bug fixes.
+
+These improvements have been across the board in all of Pulsar components,
+from new messaging features, to improved usability for Pulsar Functions
+and Pulsar IO.
+
+Check out the official <b>[release notes](/release-notes/#2.3.0)</b> for a
+detailed list of the changes, with link to the relevant pull-requests,
```
Comment (ivankelly) [inline site2/website/blog/2019-02-20-Apache-Pulsar-2-3-0.md]: typo "with links to the ..."

File: site2/website/blog/2019-02-20-Apache-Pulsar-2-3-0.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+discussions and documentation.
+
+Regarding new features introduced, I just want to highlight here a tiny
+subset between them:
```
Comment (ivankelly) [inline site2/website/blog/2019-02-20-Apache-Pulsar-2-3-0.md]: subset of them

File: site2/website/blog/2019-02-20-Apache-Pulsar-2-3-0.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+<!--truncate-->
+
+### Pulsar functions in Kubernetes
+
+It's now possible to use Kubernetes as the scheduler for Pulsar Functions.
+
+When a Pulsar cluster is configured to use Kubernetes, submitting a
+function, with the using CLI tools or REST API, will cause a
```
Comment (ivankelly) [inline site2/website/blog/2019-02-20-Apache-Pulsar-2-3-0.md]: typo. "with the CLI tools or REST API"

will cause a what?

File: site2/website/blog/2019-02-20-Apache-Pulsar-2-3-0.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+With this runtime manager, it's possible to set quota on CPU/Mem and
+have Kubernetes assign the required resources and enforce isolation
+between different instances and functions.
+
+###  New Pulsar IO connectors:
+
+A new batch of connector was added, including MongoDB, Elastic Search,
```
Comment (ivankelly) [inline site2/website/blog/2019-02-20-Apache-Pulsar-2-3-0.md]: A new batch of connectors

File: site2/website/blog/2019-02-20-Apache-Pulsar-2-3-0.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+HBase and local files source and sink.
+
+Between them there is also support for doing
```
Comment (ivankelly) [inline site2/website/blog/2019-02-20-Apache-Pulsar-2-3-0.md]: We introduce support for...

File: site2/website/blog/2019-02-20-Apache-Pulsar-2-3-0.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+builtin connector to a Pulsar cluster and start feeding data
+from a long list of supported databases like MySQL, MongoDB,
+PostgreSQL, Oracle and SQL Server.
+
+Check out the [Debezium connector](/docs/io-cdc) documentation for how
+to get started in capturing database changes.
+
+### Token Authentication
+
+Provides a very simple and secure method of authentication for Pulsar. This is based on
```
Comment (ivankelly) [inline site2/website/blog/2019-02-20-Apache-Pulsar-2-3-0.md]: Token authentication provides...

File: site2/website/blog/2019-02-20-Apache-Pulsar-2-3-0.md (hunk 3 above)
Comment (merlimat) [reply site2/website/blog/2019-02-20-Apache-Pulsar-2-3-0.md]: ups.

**Comment by merlimat** [comment]
Thanks @ivankelly, fixed the typos and broker phrases.

**Comment by merlimat** [comment]
@ivankelly I've applied your suggested fixes. Merging for now to get the website build started.

**Pull request merged**